src/drafts/MarkdownEditor/_MarkdownInput.tsx

```diff
@@ -149,7 +151,8 @@ export const MarkdownInput = forwardRef<HTMLTextAreaElement, MarkdownInputProps>
           value={value}
           onKeyDown={onKeyDown}
           disabled={disabled}
-          aria-label="Markdown value"
+          aria-label={describedBy ? undefined : 'Markdown value'}
+          aria-describedby={describedBy}
```

Comment:
If there is a description, it feels odd that we remove the label. What's the thought process behind that?

Reply:
The prop, which is coming from `MarkdownEditor`, is being used for the `fieldset`'s `aria-desbribedBy`. But this is not read by screenreader when tabbing on the page. Issue's comment box, for ex, is using this field to add an accessible name  to the Textarea, which is missing atm. There is a report after the a11y  audit mentioning that. 'MD value' is not a good accessible name. I don't want to change too much in the API of the MDEditor  so i am using the `describedBy` arg, if existing, to describe the text area. Should i use **labelledBy** instead?

I should have added this to the PR description, will do so in a moment.

Reply:
Based on the updated PR description, 

> Updated the Markdown editor to use the 'describedBy' property, if specified, for setting aria-label to the Textarea component inside markdownEditor. This is the component that is focused by screenreaders and announced

If your goal is to set a label, my recommendation would be to `aria-label` because `aria-describedby` wouldn't do the job. Voiceover on Mac will use the placeholder as label (I'm not sure if this behavior is consistent across screen readers) and only announce the description based on the user's verbosity settings. (it does not for me)

> I don't want to change too much in the API of the MDEditor so i am using the describedBy arg, if existing, to describe the text area.
> There is a report after the a11y audit mentioning that. 'MD value' is not a good accessible name. 

That is fair. However, looking at the current default of "markdown value", it seems better to add an `aria-label` prop (can make it optional to avoid breaking changes) and then fallback to the default browser behavior of using placeholder as label instead of "markdown value". 

@iansan5653 what do you think about this solution? ^

--- 

> But this (fieldset description) is not read by screenreader when tabbing on the page.

Yes, because it is not an interactive element, but you can still use a screen reader to reach the fieldset. However, doing some digging, description is not consistently read for static elements [across screen readers](https://w3c.github.io/using-aria/#p10) :( 

As an alternate to adding description to textarea, we have the option of changing the fieldset description to a label. I'm not sure which one of these is preferred, getting some guidance from the accessibility team would be useful here!